theme/theme.inc

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * @file
+ * This file contains all theme and preprocess functions.
+ */
+
+/**
+ * Implements template_preprocess_HOOK().
+ *
+ * @param array $variables
+ *   The theme template variables.
+ */
+function template_preprocess_islandora_openseadragon_viewer(array &$variables) {
+  module_load_include('inc', 'islandora_openseadragon', 'includes/utilities');
+  $library_path = libraries_get_path('openseadragon');
+  if (isset($variables['fedora_object']) && $variables['fedora_object'] !== NULL) {
+    $message = t("The parameter 'fedora_object' in theme('islandora_openseadragon_viewer') has been deprecated. As of 7.x-1.9, please update your code before the next release.");
```

Comment:
I'll update the version number. This should only display the deprecated message if the key `fedora_object` is in use. Which I think makes sense as that's the same behaviour we have elsewhere for deprecated functions and the like.